app/src/main/java/ch/epfl/favo/view/tabs/MapPage.java

```diff
@@ -124,6 +125,8 @@ public View onCreateView(
         (IFavorViewModel)
             new ViewModelProvider(requireActivity())
                 .get(DependencyFactory.getCurrentViewModelClass());
+    // CRITICAL to prevent later calling from getting null value, DO NOT DELETE IT.
+    favorViewModel.getObservedUser();
```

Comment:
Can we maybe move this to the constructor of the viewmodel? We can make it load all the currentUserInfo from teh start

Reply:
fixed